torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -1855,7 +1867,9 @@ def get_types_for_tangents(tangents):
                     all_args = unwrap_tensor_subclasses(
                         all_args, is_joint_structure=False
                     )
-                    tangents_start_idx = len(all_args) - len_tangents - len(rng_args)
+                    tangents_start_idx = (
+                        len(all_args) - len_tangents - len(rng_args) - len(bw_tokens)
```

Comment:
just as a note to myself, the full list of backward graph input args is:
`(ctx.symints, ctx.saved_tensors, tangents, rng_args, bw_tokens)`

To be extra defensive, an you assert that the computation here is gives you the same result as `len(ctx.symints) + len(ctx.saved_tensors)`?